docs/content/routing/entrypoints.md

```diff
@@ -51,199 +56,414 @@ They define the port which will receive the requests (whether HTTP or TCP).
 
-??? example "Trusting Forwarded Headers from specific IPs"
+??? info "`lifeCycle.requestAcceptGraceTimeout`"
+    
+    _Optional, Default=0s_
+    
+    Duration to keep accepting requests prior to initiating the graceful termination period (as defined by the `graceTimeOut` option).
+    This option is meant to give downstream load-balancers sufficient time to take Traefik out of rotation.
+    
+    Can be provided in a format supported by [time.ParseDuration](https://golang.org/pkg/time/#ParseDuration) or as raw values (digits).
+    
+    If no units are provided, the value is parsed assuming seconds.
+    The zero duration disables the request accepting grace period, i.e., Traefik will immediately proceed to the grace period.
+    
+    ```toml tab="File (TOML)"
+    ## Static configuration
+    [entryPoints]
+      [entryPoints.name]
+        address = ":8888"
+        [entryPoints.name.transport]
+          [entryPoints.name.transport.lifeCycle]
+            requestAcceptGraceTimeout = 42
+    ```
+    
+    ```yaml tab="File (YAML)"
+    ## Static configuration
+    entryPoints:
+      name:
+        address: ":8888"
+        transport:
+          lifeCycle:
+            requestAcceptGraceTimeout: 42
+    ```
+    
+    ```bash tab="CLI"
+    ## Static configuration
+    --entryPoints.name.address=:8888
+    --entryPoints.name.transport.lifeCycle.requestAcceptGraceTimeout=42
+    ```
 
+??? info "`lifeCycle.graceTimeOut`"
+    
+    _Optional, Default=10s_
+    
+    Duration to give active requests a chance to finish before Traefik stops.
+    
+    Can be provided in a format supported by [time.ParseDuration](https://golang.org/pkg/time/#ParseDuration) or as raw values (digits).
+    
+    If no units are provided, the value is parsed assuming seconds.
+    
+    !!! warning "In this time frame no new requests are accepted."
+    
     ```toml tab="File (TOML)"
+    ## Static configuration
+    [entryPoints]
+      [entryPoints.name]
+        address = ":8888"
+        [entryPoints.name.transport]
+          [entryPoints.name.transport.lifeCycle]
+            graceTimeOut = 42
+    ```
+    
+    ```yaml tab="File (YAML)"
+    ## Static configuration
+    entryPoints:
+      name:
+        address: ":8888"
+        transport:
+          lifeCycle:
+            graceTimeOut: 42
+    ```
+    
+    ```bash tab="CLI"
+    ## Static configuration
+    --entryPoints.name.address=:8888
+    --entryPoints.name.transport.lifeCycle.graceTimeOut=42
+    ```
+
+### ProxyProtocol
+
+Traefik supports [ProxyProtocol](https://www.haproxy.org/download/2.0/doc/proxy-protocol.txt) version 1 and 2.
+
+If proxyprotocol header parsing is enabled for the entry point, this entry point can accept connections with or without proxyprotocol headers.
+
+If the proxyprotocol header is passed, then the version is determined automatically.
+
+??? info "`proxyProtocol.trustedIPs`" 
+    
+    Enabling Proxy Protocol with Trusted IPs.
```

Comment:
```suggestion
```

Reply:
I prefer to be homogeneous with other element.
The name of the option can be long.